bentoml/deployment/aws_lambda/unzip_requirements.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import io
+import os
+import sys
+import tarfile
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger(__name__)
+
+s3_bucket = os.environ.get('BENTOML_S3_BUCKET')
+s3_prefix = os.environ.get('BENTOML_DEPLOYMENT_PATH_PREFIX')
+
+additional_pkg_dir = '/tmp/requirements'
+
+if not os.path.exists(additional_pkg_dir):
```

Comment:
is this check necessary?

Reply:
yeah. If the directory exists, we assume all of the resources are there.  This will likely happen when the function is called, while the system is warm.